File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {

        vector<int> returnVal;

        for (int i = 0; i < nums.size(); i++) {
            int nextVal = target - nums[i];

            for (int j = i + 1; j < nums.size(); j++) {
                if (nums[j] == nextVal) {
                    returnVal.push_back(i);
                    returnVal.push_back(j);
                    break;
                }
            }

            if (returnVal.size() == 2) {
                break;
            }
        }
        return returnVal;
    }
};

File: 0001-two-sum/0001-two-sum.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    for (left=0; left<nums.length; left++) {
        const numToFind = target - nums[left];
        for (right=left+1; right < nums.length; right++) {
            if (numToFind === nums[right]) {
                return [left, right];
            }
        }
    }
};